### PR TITLE
Add front page header box

### PR DESCRIPTION
**Pull request by evgiz**
Header box for front page:

Screenshot 2023-03-09 at 20 58 29

TODO add data

### CODE REVIEW

File: frontend/src/Components/SplashHeaderBox/SplashHeaderBox.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  const containerScrollSpeed = -0.1;
+  const containerTranslation = scrollY * containerScrollSpeed;
+  const containerTransform = 'translateY(' + containerTranslation + 'px)';
+
+  return (
+    <div className={classnames(styles.container, className)} style={{ transform: containerTransform }}>
+      <div className={classnames(styles.box, styles.box_left)}>
+        <h2 className={styles.title_left}>
+          <Icon icon="bx:party" />
+          Hva skjer?
```
Comment (robines) [inline frontend/src/Components/SplashHeaderBox/SplashHeaderBox.tsx:22]:
```suggestion
          {t(KEY.common_whatsup)}
```

**Review by eiriksho — APPROVED** [review]

File: frontend/src/Components/SplashHeaderBox/SplashHeaderBox.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        <p>10:13 - Some weird event</p>
+        <p>10:13 - Some weird event</p>
+        <p>10:13 - Some weird event</p>
+        <p>10:13 - Some weird event</p>
+        <p>10:13 - Some weird event</p>
+      </div>
+      <div className={classnames(styles.box, styles.box_right)}>
+        <h2 className={styles.title_right}>
+          <Icon icon="material-symbols:calendar-month-rounded"></Icon>
+          Åpningstider
```
Comment (robines) [inline frontend/src/Components/SplashHeaderBox/SplashHeaderBox.tsx:33]:
```suggestion
          {t(KEY.opening_hours)}
```

File: frontend/src/Components/SplashHeaderBox/SplashHeaderBox.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        </h2>
+        <p>10:13 - Some weird event</p>
+        <p>10:13 - Some weird event</p>
+        <p>10:13 - Some weird event</p>
+        <p>10:13 - Some weird event</p>
+        <p>10:13 - Some weird event</p>
+      </div>
+      <div className={classnames(styles.box, styles.box_right)}>
+        <h2 className={styles.title_right}>
+          <Icon icon="material-symbols:calendar-month-rounded"></Icon>
```
Comment (robines) [inline frontend/src/Components/SplashHeaderBox/SplashHeaderBox.tsx:32]:
```suggestion
          <Icon icon="material-symbols:calendar-month-rounded" />
```

**Review by robines — CHANGES_REQUESTED** [review]

**Review by emilte — APPROVED** [review]

File: frontend/src/Components/SplashHeaderBox/SplashHeaderBox.module.scss [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+}
+
+.box_left {
+  flex-basis: 70%;
+}
+
+.box_right {
+  flex-basis: 30%;
+  background-color: $blue;
```
[the remark this answers is not in the document]
Comment (evgiz) [reply frontend/src/Components/SplashHeaderBox/SplashHeaderBox.module.scss]: Fair enough, done

**Pull request merged**